neurogym/envs/annubes.py

```diff
@@ -31,14 +31,16 @@ class AnnubesEnv(TrialEnv):
             - A callable: Function that returns the duration when called.
             - A list of numbers: Random choice from the list.
             - A tuple specifying a distribution:
-                - ("uniform", min, max): Uniform distribution between min and max.
+                - ("uniform", (min, max)): Uniform distribution between min and max.
                 - ("choice", [options]): Random choice from the given options.
                 - ("truncated_exponential", [parameters]): Truncated exponential distribution.
                 - ("constant", value): Always returns the given value.
                 - ("until", end_time): Sets duration to reach the specified end time.
             The final duration is rounded down to the nearest multiple of the simulation timestep (dt).
             Note that the duration of each input and output signal is increased by this time.
             Defaults to 500.
+        iti: Inter-trial interval, or time window between sequential trials, in ms. Same format as `fix_time`.
+            Defaults to 0.
```

Comment:
same comment as for fix time in #63 . Also, should there be an option to set the intensity during the iti?

Reply:
See my answer in #63 :)

Regarding the intensity, it was never designed to be different than 0 (see also the old annubes repo)